src/plugin.js

```diff
@@ -26,6 +26,12 @@ var defaultOptions = zoomNS.defaults = {
 		sensitivity: 3
 	}
 };
+// Stores the original options of the scales
```

Comment:
This comment doesn't add much that's not covered by the variable name already. I'd remove it